formRadios.js

```diff
@@ -0,0 +1,28 @@
+// haven't used this elsewhere yet, but the idea of keeping it here is so we can pull these values in wherever we're using them, and have one source of truth for these values (instead of updating copy in multiple files)
+const maritalStatusOptions = {
+    en: [
+      {
+        value: 'Single',
+        label: 'Single',
+      },
+      {
+        value: 'Married',
+        label: 'Married',
+      },
+      {
+        value: 'Separated',
+        label: 'Separated',
+      }, 
+      {
+        value: 'Divorced',
+        label: 'Divorced',
+      }, 
+      {
+        value: 'Widowed',
+        label: 'Widowed',
+      }
+    ]
+}
+module.exports = {
+  maritalStatusOptions
+}
```

Comment:
HMMMMM.

Can we put these in the schemas file somewhere? 
That way everything to do with questions will live together.

Also, the `en`, `fr` thing I am a bit suspicious of (all of our other content will work with the translate lib right now), but happy to let this through if you think it's easier than the alternative (ie, like what we've done with error messages). 

Finally, we wouldn't want to change the value in French, only the key.

Reply:
If you wanted to keep it in its own file, you could do something like I did with the /utils file recently. (ie, move utils.js to /utils/index.js)

https://github.com/cds-snc/cra-claim-tax-benefits/pull/25/files#diff-8d6b1f9270334e293ad1f2ddec84e050